packages/fxa-content-server/app/scripts/views/survey.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*eslint-disable no-unused-vars*/
+import React, { useState, useEffect } from 'react';
+/*eslint-enable no-unused-vars*/
+import ReactDOM from 'react-dom';
+import BaseView from './base';
+
+/*eslint-disable no-unused-vars*/
+import Survey, {
+  CreateHandleIframeTask,
+} from '@fxa-react/components/Survey/index.tsx';
+/*eslint-enable no-unused-vars*/
+
+/*eslint-disable no-unused-vars*/
+const SurveyView = props => {
+  const { surveyURL } = props;
+  /*eslint-enable no-unused-vars*/
+  const [showSurvey, setShowSurvey] = useState(true);
+  const [surveyComplete, setSurveyComplete] = useState(false);
+
+  useEffect(() => {
+    const handleIframeTask = CreateHandleIframeTask(() => {
+      setSurveyComplete(true);
+      setTimeout(() => {
+        setShowSurvey(false);
+      }, 300);
+    });
+    // here we are listening for the survey complete
+    // message from surveygizmo
+
+    window.addEventListener('message', handleIframeTask, false);
+    return () => window.removeEventListener('message', handleIframeTask);
+  }, [setShowSurvey, setSurveyComplete, setTimeout]);
+
```

Comment:
This empty line seems to be in the wrong spot, if the comment above it is for the line below it.